examples/knockback/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "dependencies": {
+	"backbone.localStorage": "^1.1.3",
+	"jquery": "^2.0.0",
+	"todomvc-app-css": "^1.0.0",
+	"todomvc-common": "^1.0.1"
```

Comment:
Sindre there is an issue with that. The .editorconfig uses tabs. Sublime on save uses the .editorconfig so it keeps happening all the time!